internal/storage/service_profile.go

```diff
@@ -296,13 +296,13 @@ func (f ServiceProfileFilters) SQL() string {
 // GetServiceProfileCount returns the total number of service-profiles.
 func GetServiceProfileCount(ctx context.Context, db sqlx.Queryer, filters ServiceProfileFilters) (int, error) {
 	query, args, err := sqlx.BindNamed(sqlx.DOLLAR, `
-		select
+		select DISTINCT
 			count(distinct sp.*)
 		from
 			service_profile sp
-		left join organization_user ou
+		inner join organization_user ou
```

Comment:
Is this correct? An organization user is optional, so I believe with an `inner join`, if no org. user exists, the service-profile is not counted.

Reply:
Filter on user ID 
when an organizationID is not set and the user is not a global admin.